crates/aptos/homebrew/README.md

```diff
@@ -43,16 +47,18 @@ aptos --help
 
 Note: This guide is for developers who are trying to update the Aptos homebrew formula.
 
+You can get the latest formula here: https://github.com/Homebrew/homebrew-core/blob/master/Formula/aptos.rb
+
 Copy the `aptos.rb` file to your `homebrew` `formula` directory. For example, on macOS with an M1, this will likely be:
```

Comment:
I think this has been updated on m2 mac. Everything now is sitting under `/opt/homebrew/Library/Homebrew`